src/mise/devcontainer-feature.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "Mise",
+    "id": "mise",
+    "version": "1.0.0",
+    "description": "A devcontainer feature to install mise",
```

Description should be about the tool itself

```suggestion
    "description": "The front-end to your dev env",
```